src/sql/src/plan/statement/dml.rs

```diff
@@ -261,8 +261,7 @@ pub fn plan_explain(
     }: ExplainStatement<Aug>,
     params: &Params,
 ) -> Result<Plan, PlanError> {
-    let is_view = matches!(explainee, Explainee::View(_));
```

Comment:
I'm removing most of this code in my next PR, I'm not sure if the changes here are needed.

See this code:

https://github.com/aalexandrov/materialize/blob/issue_20652_wip/src/sql/src/plan/statement/dml.rs#L286-L327

Reply:
Ok, you can remove it in your PR.

Btw. when you implement `EXPLAIN CREATE MATERIALIZED VIEW`, then the changes in my PR will become relevant again, in the sense of keeping in mind the following things:
- When calling `plan_root_query`, set the `QueryLifetime` to `MaterializedView`.
- No need to turn the finishing into a TopK, because `plan_root_query` will do that automatically after my PR.